ocr-numbers.md

```diff
@@ -1,36 +1,74 @@
-## Step 1
+# Step One
 
-A simple binary font has been constructed using only pipes and
-underscores.
+To begin with, convert a simple binary font to a string containing 0 or 1.
 
-The number is four rows high, three columns wide:
+The binary font uses pipes and underscores, four rows high and three columns wide.
 
+```
      _   #
     | |  # zero.
     |_|  #
          # the fourth row is always blank
+```
 
+Is converted to "0"
+
+```
          #
       |  # one.
       |  #
          # (blank fourth row)
+```
+
+Is converted to "1"
+
+If the input is the correct size, but not recognizable, your program should return '?'
+
+If the input is the incorrect size, your program should return an error.
+
+# Step Two
 
-Write a program that, given a 3 x 4 grid of pipes, underscores, and
-spaces, can determine whether the the grid represents a zero, a one, or
-garble.
+Update your program to recognize binary strings, replacing garbled numbers with ?
```

Comment:
hmm, maybe it's just me, but I guess it might benefit from being explicit that there can be more than one character now.